react/components/enhancers/options.js

```diff
@@ -1,33 +1,55 @@
 export const SHAPES = [
-    {
-        id: 'square',
-        modifier: 'br1'
-    },
-    {
-        id: 'rounded',
-        modifier: 'br3'
-    },
-    {
-        id: 'circle',
-        modifier: 'br-100'
-    }
+  {
+    id: 'square',
+    modifier: 'br1'
+  },
+  {
+    id: 'rounded',
+    modifier: 'br3'
+  },
+  {
+    id: 'circle',
+    modifier: 'br-100'
+  }
 ]
 
 export const ORIENTATIONS = [
-    {
-      id: 'up',
-      modifier: '--up',
-    },
-    {
-      id: 'down',
-      modifier: '--down',
-    },
-    {
-      id: 'right',
-      modifier: '--right',
-    },
-    {
-      id: 'left',
-      modifier: '--left',
-    },
+  {
+    id: 'up',
+    modifier: '--up',
+  },
+  {
+    id: 'down',
+    modifier: '--down',
+  },
+  {
+    id: 'right',
+    modifier: '--right',
+  },
+  {
+    id: 'left',
+    modifier: '--left',
+  },
+]
+
+export const STATES = [
+  {
+    id: 'on',
+    modifier: '--on',
+  },
+  {
+    id: 'off',
+    modifier: '--off',
+  },
+]
+
+export const TYPES = [
+  {
+    id: 'filled',
+    modifier: '-f',
```

Comment:
This modifier could be changed on store to --filled. And be a suffix, as the others.